src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -378,16 +375,17 @@ const formConfig = {
           uiSchema: studentLastTerm.uiSchema,
           schema: studentLastTerm.schema,
         },
-        // NOTE: These are temporarily disabled, and will be reintroduced post-launch as part of 674 pension support.
         studentIncomeInformation: {
-          depends: () => false,
+          depends: formData =>
+            isChapterFieldRequired(formData, TASK_KEYS.report674),
           title: 'Information needed to add a student 18 to 23 years old',
           path: 'report-674-student-income-information',
           uiSchema: studentIncomeInformation.uiSchema,
           schema: studentIncomeInformation.schema,
         },
         studentNetworthInformation: {
-          depends: () => false,
```

Comment:
These `depends()` need to stay since we're releasing these two pages after launch now.

Reply:
same as above, fixed these.